BinaryObjectScanner/FileType/InstallShieldCAB.cs

```diff
@@ -24,6 +24,16 @@ public bool Extract(string file, string outDir, bool includeDebug)
         /// <inheritdoc/>
         public bool Extract(Stream? stream, string file, string outDir, bool includeDebug)
         {
+            try
+            {
+                file = Path.GetFullPath(file);
+            }
+            catch
+            {
+                // If any of the exceptions to getting a full path occur, just continue as normal. The rest of the
+                // functions already handle the various cases upon which this would throw exceptions.
+            }
```

Comment:
If you have a path that throws on getting the full path, then the rest of this will likely fail anyway.